typing/printtyp.mli

```diff
@@ -94,13 +94,17 @@ val type_scheme_max: ?b_reset_names: bool ->
 (* End Maxence *)
 val tree_of_value_description: Ident.t -> value_description -> out_sig_item
 val value_description: Ident.t -> formatter -> value_description -> unit
+val label : formatter -> label_declaration -> unit
+val constructor : formatter -> constructor_declaration -> unit
 val tree_of_type_declaration:
     Ident.t -> type_declaration -> rec_status -> out_sig_item
 val type_declaration: Ident.t -> formatter -> type_declaration -> unit
 val tree_of_extension_constructor:
     Ident.t -> extension_constructor -> ext_status -> out_sig_item
 val extension_constructor:
     Ident.t -> formatter -> extension_constructor -> unit
+val extension_only_constructor:
+    Ident.t -> formatter -> extension_constructor -> unit
```

Comment:
Last nitpicking: the difference between `extension_constructor` and `extension_only_constructor` should be documented.

Reply:
Added comments with output examples.

Reply:
Perfect, thanks.